pay-api/src/pay_api/services/payment_account.py

```diff
@@ -414,12 +414,13 @@ def find_by_id(cls, account_id: int):
 
     @staticmethod
     def _calculate_activation_date():
-        """Find the activation date."""
+        """Find the activation date in local time.Convert it to UTC before returning."""
         account_activation_wait_period: int = current_app.config.get('PAD_CONFIRMATION_PERIOD_IN_DAYS')
-        date_after_wait_period = datetime.today() + timedelta(account_activation_wait_period)
+        date_after_wait_period = current_local_time() + timedelta(account_activation_wait_period)
```

Comment:
looks like timedelta is missing days argument?

Reply:
timedelta defaults to days.so it shud work..